src/test-util/mock-imported-components.js

```diff
@@ -38,7 +49,8 @@ function getDisplayName(component) {
 
 /**
  * Helper for use with `babel-plugin-mockable-imports` that mocks components
- * imported by a file.
+ * imported by a file. This will only mock components that are local to the
+ * package; it will not mock third-party components.
```

Comment:
I think we need to say something here about why we're making this distinction between local and third-party components. Perhaps something along these lines:

We believe that in terms of balancing the competing tensions between keeping unit tests fast and isolated from the implementation details of other modules, and helping to catch integration issues when changes occur in those other modules, we think it is a better balance not to mock components from external modules by default. This is also consistent of non-UI code from external packages. The code which sits at the boundary between the client and the package tends not to mock that package.

Reply:
I've updated the comment; it's still a little terse. Let me know if you have any specific suggestions; I'm feeling some writer's block.